src/commands/pressable-site-php-errors.php

```diff
@@ -238,7 +238,7 @@ private function analyze_error_entries( array $parsed_php_errors ): array {
 		foreach ( $parsed_php_errors as $parsed_php_error ) {
 			$error_hash = \hash( 'md5', $parsed_php_error['error_message'] );
 			if ( isset( $stats_table[ $error_hash ] ) ) {
-				$stats_table[ $error_hash ]['error_count']++;
+				++$stats_table[ $error_hash ]['error_count'];
```

Comment:
The only difference between `++$foo` and `$foo++` is whether it returns `$foo` or `$foo + 1`, right?  I'm not seeing us doing anything with the return value here, so I'm not quite sure what this change impacts..

Reply:
That was a change that `phpcbf` did ... since this command is part of the Pressable group of commands that this PR targets, I thought it made sense to leave the change.